src/Docnet.Core/Validation/PageRangeValidator.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+
+namespace Docnet.Core.Validation
+{
+    public class PageRangeValidator : IPageRangeValidator
```

Comment:
As mentioned above it def should not have an interface and for consistency sake it should be with other validation methods.

Reply:
moved to `Validator`